src/rust/perftools/profiler/scope.rs

```diff
@@ -36,6 +36,8 @@ pub struct Scope {
     pub num_calls: usize,
     /// In total, how much time has been spent in this scope?
     pub duration_sum: u64,
+    /// In total, how much time has been spent in this scope only (excluding chlid scopes)?
```

Comment:
It seems a little convoluted to store the exclusive duration sum here just to print it. We should probably compute it and directly return it for printing without storing it in the scope.